spec/system/consumer/authentication_spec.rb

```diff
@@ -156,6 +156,27 @@
       end
     end
 
+    describe "Logging in from the private shop page" do
+      let(:distributor) { create(:distributor_enterprise, require_login: true) }
+      let!(:order_cycle) {
+        create(:simple_order_cycle, distributors: [distributor],
+                                    coordinator: create(:distributor_enterprise))
+      }
+      before do
+        visit enterprise_shop_path(distributor)
+      end
+
+      it "clicking login triggers the login modal" do
+        find("a", text: "login").click
```

Comment:
We might need to be a bit more careful here, as the navigation bar will also have an `<a>` tag that has text: "Login". Is the matcher case-sensitive? Maybe this should be wrapped in a `within "#shop-tabs" do` block?